docs/pvacview/pvacseq_module/pvacseq_vignette.rst

```diff
@@ -0,0 +1,382 @@
+- A json file of Class I data: JSON file with detailed information about the predicted epitopes, formatted for pVACview.
+- A tsv for Class II data: an aggregated file that lists all MHC-class-II predicted epitopes. 
+- A list of genes of interest: in this example, it's a tsv of the cancer census hotspot gene list from COSMIC. 
+
+
+.. figure:: ../../images/screenshots/vignette/pvacview-loadDemoDataset.png
+    :width: 1000px
+    :align: right
+    :alt: pVACview Vignette
+    :figclass: align-left
+
+:large:`Visualize and Explore`
+______________________________
+
+You will next be taken to a view similar to this.
+
+.. figure:: ../../images/screenshots/vignette/pvacview-mainviewDemodataset.png
+    :width: 1000px
+    :align: right
+    :alt: pVACview Vignette
+    :figclass: align-left
+
+The top row of the page has 4 sections: 
+
+- Advanced Options: Regenerate Tiering with different parameters
+- Original Parameters for Tiering
+- Current Parameters for Tiering
+- Add Comments for selected variant
+
+pVACview prioritizes neoantigen candidates by ranking these peptides based on a set of rules (parameters for tiering), which include variant allele fraction cutoff, gene/transcript expression, binding affinity predictions and more as discussed later. Based on routine criteria described in the literature, we provide a default set of parameters for tiering, detailed in the **Original Parameters for Tiering** section. The default is a good starting point, but as all samples are unique in terms of sample quality, sequencing quality, tumor purity, tumor mutation burden, HLA type, you may also want to set your own parameters in the **Advanced Options: Regenerate Tiering with different parameters** section. To see the real-time set of rules applied to your data, see the **Current Parameters for Tiering** section.
+Note: click the ``+``/ ``-`` in the right corner to expand/contract each section.
+
+The original parameters rank candidates on multiple facets.
+The first aspect is clonality. Cancer starts with a founding clone with tumor-initiating mutations which expand and drive malignancy. Descendents of the founding clone may acquire additional mutations. Clonal mutations are shared by all clones, whereas subclonal mutations are shared by some but not all cancer cells. Neoantigen candidates derived from clonal variants should be prioritized as it has been proposed that targeting such mutations will drive a better clinical response. pVACview uses the following parameters when determining clonality:
+
+- ``Tumor Purity`` : a value between 0 and 1 indicating the fraction of tumor cells in the tumor sample. (default: None)
+- ``VAF Clonal`` : Tumor DNA variant allele frequency (VAF) to determine whether the variant is clonal. This value is determined automatically from the VAFs in the input data (see pVACtools docs for further details) but can be adjusted by the user (see below).
```

Comment:
```suggestion
- ``VAF Clonal`` : Tumor DNA variant allele frequency (VAF) to determine whether the variant is clonal. By default, this value is determined automatically from the VAFs in the input data during the original pVACseq run unless the tumor purity parameter is set (see pVACseq docs for further details). This can be adjusted by the user in pVACview (see below).
```